ignite/pkg/nodetime/programs/sta/sta.go

```diff
@@ -66,13 +67,74 @@ func Generate(ctx context.Context, outPath, specPath string, options ...Option)
 	dir := filepath.Dir(outPath)
 	file := filepath.Base(outPath)
 
+	// generate temp template directory
+	templateTmpPath, err := os.MkdirTemp("", "gen-js-sta-templates")
+	if err != nil {
+		return err
+	}
+	routeNameTemplate := `<%
```

Comment:
This variable could be a constant defined at the top of the file instead of inside the `Generate` function:
```go
const routeNameTemplate = `<%
 ...
%>`
```